### PR TITLE
UX-275/Modified payload for scaling master and worker nodes

### PR DESCRIPTION
**Pull request by knnguy**
- Payload for scaling worker nodes has been modified

Screen Shot 2020-09-17 at 3 29 31 PM

### CODE REVIEW

**Comment by knnguy** [comment]
@jruddell I noticed that the `handleSubmit` function is passed to the `ScaleMasters` component in `ScaleMastersPage.tsx` , but it is never used. I still modified the `handleSubmit` function anyways, but not sure if that should be taken out.

Screen Shot 2020-09-17 at 4 15 12 PM

**Comment by jruddell** [comment]
> @jruddell I noticed that the `handleSubmit` function is passed to the `ScaleMasters` component in `ScaleMastersPage.tsx` , but it is never used. I still modified the `handleSubmit` function anyways, but not sure if that should be taken out.
> 
> Screen Shot 2020-09-17 at 4 15 12 PM

I believe when Brennan wrote this the intention was to have the ability to pass an onSubmit prop to this component to override how we normally submit... Should the view need to override it. But if were not using it then we can omit for now.

File: src/app/plugins/kubernetes/components/infrastructure/clusters/actions.js [hunk 1]
```diff
@@ -80,7 +80,6 @@ export const clusterActions = createCRUDActions(ActionDataKeys.Clusters, {
     const updateableParams = 'name tags numWorkers numMinWorkers numMaxWorkers'.split(' ')
 
     const body = pick(updateableParams, params)
-    body.etcdBackup = getEtcdBackupPayload('etcdBackup', params)
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/infrastructure/clusters/actions.js:83]: Does this break regular cluster updates? did you test updating a cluster and changing etcd fields?

Comment (jruddell) [reply src/app/plugins/kubernetes/components/infrastructure/clusters/actions.js:83]: Maybe we should keep this line but wrap it in a conditional so we attach it to the payload when needed

**Comment by knnguy** [comment]
etcdBackup is only sent in the payload when needed

Screen Shot 2020-09-21 at 1 05 50 PM

**Review by jruddell — APPROVED** [review]

**Pull request merged**